model/system/metrics.yaml

```diff
@@ -394,3 +394,16 @@ groups:
     unit: "By"
     attributes:
       - ref: linux.memory.slab.state
+
+  # system.uptime metric
+  - id: metric.system.uptime
+    type: metric
+    metric_name: system.uptime
+    stability: experimental
+    brief: "The time the system has been running"
```

Comment:
@open-telemetry/specs-semconv-maintainers My instinct is to say that the brief should mention that this is in seconds in particular; different OS APIs may provide a value in seconds or milliseconds so it feels like a good idea to be specific in the description on top of `s`. But I'm not sure if there's a stance against being redundant about units in descriptions.

Reply:
No stance, it's ok to repeat something if it's important.

If we happen to mention unit in the brief it'd be nice to also mention that it's floating point.